tests/testthat/test-run_pipeline.R

```diff
@@ -18,7 +18,8 @@ test_that("run_pipeline errors for unsupported method", {
     run_pipeline(
       otu_small,
       "not_a_method"
-    )
+    ),
+    "Method 'not_a_method' is not supported. Supported methods are:"
```

Comment:
Do we have to list what the supported methods are here?

Reply:
It's a regex, so it matches the first part of the error message up to the point where it actually lists them.